pkg/synthetictests/kubelet.go

```diff
@@ -135,6 +135,26 @@ func testContainerFailures(events monitorapi.Intervals) []*ginkgo.JUnitTestCase
 	return testCases
 }
 
+func testKubeApiserverProcessOverlap(events monitorapi.Intervals) []*ginkgo.JUnitTestCase {
+	const testName = "[sig-node] overlapping apiserver process detected during kube-apiserver rollout"
+	success := &ginkgo.JUnitTestCase{Name: testName}
+	failures := []string{}
+	for _, event := range events {
+		if strings.Contains(event.Message, "reason/TerminationProcessOverlapDetected") {
+			failures = append(failures, event.Locator)
+		}
+	}
+
+	failure := &ginkgo.JUnitTestCase{
+		Name:      testName,
+		SystemOut: strings.Join(failures, "\n"),
+		FailureOutput: &ginkgo.FailureOutput{
+			Output: fmt.Sprintf("The following events detected:\n\n%s", strings.Join(failures, "\n")),
```

Comment:
you're going to want to display the time as well.

Reply:
fixed